src/fail.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+
+const { toMultipleTestResults } = require("./toMultipleTestResults");
+
+module.exports.fail = ({ start, end, failures }) =>
```

Comment:
We can probably use brackets, makes it easier to read :)

Reply:
Not very sure of this means. Can you please suggest the change

Reply:
Ehm, I don't know how to lay down that in words. I will make the change myself in this PR. Don't worry.

Reply:
Ok